pkg/interfaces/contracts/standalone-utils/ILPOracleFactoryBase.sol

```diff
@@ -36,6 +36,10 @@ interface ILPOracleFactoryBase {
 
     /**
      * @notice Creates a new oracle for the given pool.
+     * @dev Note that the caller must ensure that the given `feeds` are correct for the corresponding pool tokens.
+     * The contract checks that the array lengths match, but cannot independently verify correctness. See the docs for
+     * notes on how to pair feeds (and rate providers) with tokens, as there are many subtleties.
```

Comment:
Should we also clarify that oracles can be recreated with the right feed / token order in case there's a mistake?

Reply:
Sure, I can add that.